app/service/data_svc.py

```diff
@@ -160,7 +161,8 @@ async def load_ability_file(self, filename, access):
                 requirements = await self._load_ability_requirements(ab.pop('requirements', []))
                 buckets = ab.pop('buckets', [tactic])
                 ab.pop('access', None)
-                plugin = ab.pop('plugin', filename.split('/')[1])
+                plugin_path = pathlib.PurePath(filename).parts
+                plugin = plugin_path[1] if 'plugins' in plugin_path else None
```

Comment:
Might be worth pulling this out to its own function since the same code is used in `load_yaml_file()`